packages/history-utility/src/index.ts

```diff
@@ -134,7 +137,7 @@ export function proxyWithHistory<V>(
      *   - history.wip: field for holding sandbox changes; used to avoid infinite loops<br>
      */
     history: ref<History<V>>({
-      wip: undefined, // to avoid infinite loop
+      wip: EmptyWIP, // to avoid infinite loop
```

Comment:
also ensure the others are renamed

```suggestion
      wip: EMPTY_WIP, // to avoid infinite loop
```